### ISSUE TITLE
Not sure how to run the decrypted file

### ISSUE THREAD

**Issue by rupert-adams** [mills5/quakeat-decrypted#7]
Hi there,

I've downloaded the quakeat.chd and the glquake_decrypted.exe file and I'm not sure how to actually run this. I have Mame but all the guides I've read on .chd files have suggested I need the rom file as well. I cannot find the rom file for quakeat. So how do I set up and run the game? 

Any help will be greatly appreciated!

**Comment by Neo-Desktop** [comment]
quakeat.chd is a full harddrive copy of a windows 95 system - you should be able to convert it to something like Virtualbox and run it that way or extract the necessary files and run it under modern windows

**Comment by rupert-adams** [comment]
ohhhh cool! and I add the .exe to the Vbox instance?

**Comment by Neo-Desktop** [comment]
Yep, that's correct, though bear in mind that virtualbox may not work due to a lack of drivers. I highly recommend running it on a more modern version of windows.

Once converted from a chd file to a raw image file (about 2gb in size) you can navigate the filesystem using 7zip and extract the quake directory to a folder elsewhere on your host system. Then just copy the patched exe in that directory and make the other changes as specified in the readme. You should be able to play it as intended at this point

**Comment by rupert-adams** [comment]
Thank you for your help with this, last question: I am inexperienced with Windows (I work with Unix systems usually). What would you recommend using to convert the image from chd to raw?

**Comment by Neo-Desktop** [comment]
Take a look at [chdman(1)](http://manpages.ubuntu.com/manpages/bionic/man1/chdman.1.html)

`chdman extracthd` or `chdman extractraw` should work. I can't remember off the top of my head what I used, but those are the relevant options.

You may be able to run quake under wine, I'm not 100% sure though. Your mileage may vary
